utilities/transactions/optimistic_transaction_test.cc

```diff
@@ -1269,61 +1205,52 @@ TEST_P(OptimisticTransactionTest, UndoGetForUpdateTest) {
   ReadOptions read_options, snapshot_read_options;
   OptimisticTransactionOptions txn_options;
   string value;
-  Status s;
 
-  txn_db->Put(write_options, "A", "");
+  ASSERT_OK(txn_db->Put(write_options, "A", ""));
 
   Transaction* txn1 = txn_db->BeginTransaction(write_options);
   ASSERT_TRUE(txn1);
 
-  s = txn1->GetForUpdate(read_options, "A", &value);
-  ASSERT_OK(s);
+  ASSERT_OK(txn1->GetForUpdate(read_options, "A", &value));
 
   txn1->UndoGetForUpdate("A");
 
   Transaction* txn2 = txn_db->BeginTransaction(write_options);
   txn2->Put("A", "x");
-  s = txn2->Commit();
-  ASSERT_OK(s);
+  ASSERT_OK(txn2->Commit());
   delete txn2;
 
   // Verify that txn1 can commit since A isn't conflict checked
-  s = txn1->Commit();
-  ASSERT_OK(s);
+  ASSERT_OK(txn1->Commit());
   delete txn1;
 
   txn1 = txn_db->BeginTransaction(write_options);
-  txn1->Put("A", "a");
+  ASSERT_OK(txn1->Put("A", "a"));
 
-  s = txn1->GetForUpdate(read_options, "A", &value);
-  ASSERT_OK(s);
+  ASSERT_OK(txn1->GetForUpdate(read_options, "A", &value));
 
   txn1->UndoGetForUpdate("A");
 
   txn2 = txn_db->BeginTransaction(write_options);
-  txn2->Put("A", "x");
-  s = txn2->Commit();
-  ASSERT_OK(s);
+  ASSERT_OK(txn2->Put("A", "x"));
+  ASSERT_OK(txn2->Commit());
   delete txn2;
 
   // Verify that txn1 cannot commit since A will still be conflict checked
-  s = txn1->Commit();
+  Status s = txn1->Commit();
   ASSERT_TRUE(s.IsBusy());
   delete txn1;
 
   txn1 = txn_db->BeginTransaction(write_options);
 
-  s = txn1->GetForUpdate(read_options, "A", &value);
-  ASSERT_OK(s);
-  s = txn1->GetForUpdate(read_options, "A", &value);
-  ASSERT_OK(s);
+  ASSERT_OK(txn1->GetForUpdate(read_options, "A", &value));
+  ASSERT_OK(txn1->GetForUpdate(read_options, "A", &value));
 
```

Comment:
Not arguing that the code is not equivalent to what it was before, but is there a reason for the two identical calls back to back?

Reply:
Removing the second call would cause the test to fail.